doc/manpages/qvm-service.rst

```diff
@@ -137,9 +137,32 @@ disable-dns-server
     Enabling this service will result in an empty /etc/resolv.conf.
     The functionality is implemented in /usr/lib/qubes/setup-ip.
 
+lightdm:
+    Default: enabled in GPU and VNC GuiVM
+
+    Start `lightdm` on boot which prevent `qubes-gui-agent` to start.
```

Comment:
Those are rather two separate effects of this service: Start `lightdm` and avoid starting `qubes-gui-agent`.